### PR TITLE
more turbo lint attempts

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
wip

### CODE REVIEW

File: turbo.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
       "cache": false
     },
     "lint": {
+      "dependsOn": ["^build", "codegen:bigBoi", "codegen:bigLegacyRouter"],
```
Comment (juliusmarminge) [inline turbo.json:37]: this won't do anything unless we run the lint script with turbo

Comment (JoshuaKGoldberg) [reply turbo.json:37]: Are we not? Or is there a script that's more turbo-hooked-up?

Comment (juliusmarminge) [reply turbo.json:37]: No, ESLint is currently just being run on its own:

https://github.com/trpc/trpc/blob/db189c9aa569f0f066ccd432a83920093fa2d6a7/package.json#L15

The `turbo-lint` PR (#3186) was made to split this up and run them all using Turbo

Comment (juliusmarminge) [reply turbo.json:37]: Oh waiiit i checked the wrong branch... In here it's totally valid. Thought you were based off of main 🤦🏼‍♂️

**Pull request closed without merging**